testutils/gradle/testing.gradle

```diff
@@ -23,7 +23,7 @@ dependencies {
     testCompile "org.spockframework:spock-core:${spock_version}"
     testCompile "org.spockframework:spock-spring:${spock_version}"
     testCompile "org.springframework.batch:spring-batch-test:${spring_batch_version}"
-    testCompile "org.springframework:spring-test:${spring_version}"
+    testCompile "org.springframework:spring-test:3.2.9.RELEASE"
```

Comment:
Should this be `3.2.9.RELEASE_1` ? (from https://github.com/TWCable/grabbit/pull/53/files#diff-6512f838e273b79676cac5f72456127fR1 )